.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Is this file needed? Otherwise maybe put it in `.gitignore`

Reply:
i will delete it.